inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/group/InlongGroupInfo.java

```diff
@@ -116,6 +116,12 @@ public abstract class InlongGroupInfo {
     @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
     private Date modifyTime;
 
+    @ApiModelProperty(value = "Backup inlong cluster tag")
+    private String backupInlongClusterTag;
```

Comment:
If this parameter is needed in all group information, why not add a column to store it in the database instead of putting it in extParams?

Reply:
good suggestion, have added them to database